### PR TITLE
[doc][yugabyted] Set preferred region

### PR DESCRIPTION
**Pull request by ddhodge**
Set preferred regions in yugabyted. Format.
DOC-382

@netlify /preview/reference/configuration/yugabyted/

### CODE REVIEW

**Review by nchandrappa — APPROVED** [review]
LGTM.

File: docs/content/preview/reference/configuration/yugabyted.md [hunk 1]
```diff
@@ -426,7 +415,7 @@ For example, to create a new read replica cluster, execute the following command
 
 #### modify
 
-Use the subcommand `yugabyted configure_read_replica modify` to modify an exisitng read replica cluster.
+Use the sub-command `yugabyted configure_read_replica modify` to modify an existing read replica cluster.
 
 For example, modify a read replica cluster using the following commands.
```
Comment (aishwarya24) [inline docs/content/preview/reference/configuration/yugabyted.md:420]: indent the code blocks under the bullets.

File: docs/content/preview/architecture/key-concepts.md [hunk 2]
```diff
@@ -145,7 +168,7 @@ Tablets are also referred as shards.
 
 ## Tablet leader
 
-In a cluster, each [tablet](#tablet) is replicated as per the [replication factor](#replication-factor-rf) for high availability. Amongst these tablet replicas one tablet is elected as the leader and is responsible for handling writes and consistent reads. The other replicas as termed followers.
+In a cluster, each [tablet](#tablet) is replicated as per the [replication factor](#replication-factor-rf) for high availability. Amongst these tablet replicas one tablet is elected as the leader and is responsible for handling writes and consistent reads. The other replicas are called followers.
```
Comment (aishwarya24) [inline docs/content/preview/architecture/key-concepts.md:171]: I think we should either add a separate entry for tablet followers or change the title to Tablet leader and followers.

**Review by aishwarya24 — APPROVED** [review]
LGTM. Copy to stable.

**Pull request merged**